price-feeder/oracle/oracle.go

```diff
@@ -276,46 +276,56 @@ func GetComputedPrices(
 	providerPairs map[string][]types.CurrencyPair,
 	deviations map[string]sdk.Dec,
 ) (prices map[string]sdk.Dec, err error) {
-	// filter out any erroneous candles
-	filteredCandles, err := FilterCandleDeviations(
+	// convert any non-USD denominated candles into USD
+	convertedCandles, err := convertCandlesToUSD(
 		logger,
 		providerCandles,
+		providerPairs,
 		deviations,
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	// convert any non-USD denominated candles into USD.
-	convertedCandles, err := convertCandlesToUSD(filteredCandles, providerPairs)
+	// filter out any erroneous candles
+	filteredCandles, err := FilterCandleDeviations(
+		logger,
+		convertedCandles,
+		deviations,
+	)
```

Comment:
If we have less than 3 values, then we won't have a price. Is this desirable?

Reply:
I don't believe this is true - with 2 or less values we don't filter anything out, because we can't calculate a meaningful standard deviation. Of course the best case is having 3+ values.

Reply:
https://github.com/umee-network/umee/blob/woz/fix-multi-quote-filtering/price-feeder/oracle/util.go#L153

Reply:
https://github.com/umee-network/umee/blob/4348c3e433df8c37dd98a690e96fc275de609bc1/price-feeder/oracle/filter.go#L115

Reply:
How is this related? Std dev is well defined even for single data point.